contracts/solidity/.env.example

```diff
@@ -1,4 +1,7 @@
+# To deploy Ethereum contract 
 ETH_RPC_URL=<ethereum_rpc_url>
 ETH_PRIVATE_KEY=<eth_private_key> #in lowercase hexa with the 0x prefix
 ETHERSCAN_API_KEY=<etherscan_api_key>
 SN_MESSAGING_ADDRESS=<0xde29d060D45901Fb19ED6C6e959EB22d8626708e|0xE2Bb56ee936fd6433DC0F6e7e3b8365C906AA057|0xc662c410C0ECf747543f5bA90660f6ABeBD9C8c4> # Goerli | Sepolia | Mainnet
+# To upgrade Ethereum contract 
+YAB_TRANSFER_PROXY_ADDRESS=<YAB_TRANSFER_PROXY_ADDRESS>
```

Comment:
This address could be automatically set by the deploy script.
Something similar is done here:
https://github.com/yetanotherco/yet-another-bridge/blob/main/contracts/cairo/deploy.sh#L44

Reply:
Wait I'm not sure this is even necessary because the address is already saved. We do it with the setter after the deployment in Starknet right?

Reply:
Done, now when you deploy a contract, that variable is automatically set